Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,19 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import "./Header.css";
+import ImageThumbnail from "./ImageThumbnail";
+import HeaderTitle from "./HeaderTitle";
+import HeaderContent from "./HeaderContent";
+
+const HeaderContainer = () => {
+  return (
+    <div className="header-container">
+      <ImageThumbnail />
+      <div className="header-text">
+        <HeaderTitle />
+        <HeaderContent />
+      </div>
```

Comment:
wrapping the title and content components here is perfect, since it allows you to apply some flexbox magic to get your desired behavior